MiniApp/Classes/core/MiniAppSdkConfig.swift

```diff
@@ -81,6 +81,14 @@ public class MiniAppSdkConfig {
         }
     }
 
+    public var dynamicDeepLinksList: [String]? {
+        didSet {
+            if dynamicDeepLinksList?.count == 0 {
```

Comment:
This should not give the expected result, you should try this:

```suggestion
            if dynamicDeepLinksList?.count ?? 0 == 0 {
```

or 

```suggestion
            if dynamicDeepLinksList == nil {
```